packages/dbml-connector/src/connectors/mssqlConnector.ts

```diff
@@ -49,19 +49,51 @@ const getValidatedClient = async (connection: string): Promise<sql.ConnectionPoo
 
 const convertQueryBoolean = (val: string | null): boolean => val === 'YES';
 
-const getFieldType = (data_type: string, default_type: DefaultType, character_maximum_length: number, numeric_precision: number, numeric_scale: number): string => {
+const getFieldType = (data_type: string, character_maximum_length: number, numeric_precision: number, numeric_scale: number): string => {
   if (MSSQL_DATE_TYPES.includes(data_type)) {
     return data_type;
   }
+
+  // timestamp is a synonym of rowversion
+  // microsoft recommends that we should not use timestamp, so we will convert it back to rowversion
+  // https://learn.microsoft.com/en-us/sql/t-sql/data-types/rowversion-transact-sql?view=sql-server-ver15
+  if (data_type === 'timestamp') {
+    return 'rowversion';
+  }
```

Comment:
I don't think so, we should use the same `data_type` that match exactly the user's database nha.

Reply:
got it anh, I will return `timestamp` instead, and notice that we cannot use syntax `timestamp(n)`

Reply:
sure, we do not support `datetime_type(n)`